src/views/QuestionView.vue

```diff
@@ -242,16 +238,33 @@ export default {
             }
           });
           if (
-            result.data.questions.length < count &&
+            result.data.count < count &&
             (clean || !this.questionBuffer.includes(NO_QUESTION_LEFT))
           ) {
             dataToAdd.push(NO_QUESTION_LEFT);
           }
+          if (this.page == Math.ceil(this.remainingQuestionCount / count)) {
+            if (!this.questionBuffer.includes(NO_QUESTION_LEFT)) {
+              if (clean) {
+                this.questionBuffer = [NO_QUESTION_LEFT];
+              } else {
+                this.questionBuffer = [...this.questionBuffer, ...dataToAdd];
+                this.questionBuffer = [
+                  ...this.questionBuffer,
+                  NO_QUESTION_LEFT,
+                ];
```

Comment:
You can do all at once
```suggestion
                this.questionBuffer = [
                  ...this.questionBuffer,
                   ...dataToAdd,
                  NO_QUESTION_LEFT,
                ];
```